lib/organisms/widgets-area/widgets-area.directive.js

```diff
@@ -36,4 +38,23 @@ function lnOWidgetsArea($compile) {
 
     $compile(elem.contents())(scope);
   }
+
+  function getWidgetType(type) {
+    switch (type) {
+      case 'lean-downloads':
```

Comment:
Yes.. Let me work on that because it requires renaming for some widgets:

- ln-m-widget-social must be renamed to ln-m-widget-social-links
- ln-m-widget-contact must be renamed to ln-m-widget-contact-details